cardano-api/src/Cardano/Api/ProtocolParameters.hs

```diff
@@ -312,7 +312,7 @@ instance FromJSON ProtocolParameters where
         <*> o .: "poolPledgeInfluence"
         <*> o .: "monetaryExpansion"
         <*> o .: "treasuryCut"
-        <*> o .:? "utxoCostPerWord"
```

Comment:
It would work but it would be confusing when looking at the `ProtocolParameters` type. It's better to make the changes obvious rather than having it only at the JSON deserialization level.